understand/observed-size-spectra.qmd

```diff
@@ -319,15 +318,15 @@ Fitting a same linear model to the binned biomass density data now gives
 (model <- lm(log(Biomass_dens) ~ log(mid_points[bin]), data = binned_biomass))
 ```
 
-The slope is `r signif(model$coefficients[[2]], 4)` for the biomass density whereas it was `r exponent` for the number density. This makes perfect sense, because if we denote the biomass density by $B(w)$ then $B(w) = w N(w)$. Therefore if numbers at weight scale with the exponent $\lambda$ as $N(w) \propto w^{-\lambda}$ then biomass at weight will scale as $B(w)\propto w^{1-\lambda}$.
```

Comment:
We should be really careful not to confuse numbers with number densities. That is one of the main messages we are trying to get across.